sigs/sig-marketing/README.md

```diff
@@ -18,6 +18,10 @@ Sig marketing meetings are held twice monthly. Meetings can be found in the [Arg
 | Member | Create and approve official communications (emails, social, blogs, etc). Manage and work on official events. Work in committee on projects as deemed appropriate by ArgoProj maintainers. Provide regular updates to maintainers on marketing initiatives. |
 | Interested Party | Provide support, create content, and assist with sig-marketing initiatives. No nomination is required to be added as an interested party. |
 | Lead | Coordinates work in the SIG. Leads and runs the SIG Marketing meetings. Reports back on SIG Marketing completed and upcoming work to maintainers. |
+| Emeritus | Inactive members that can eventually become active again (by submitting a PR) without the need for additional processes.
```

Comment:
Argo Proj member or SIG member? I think process for inactive members is already documented in membership doc.

Reply:
This is only applicable to the scope of the SIG marketing.